manifests/vrrp/instance.pp

```diff
@@ -162,17 +176,20 @@
   $virtual_ipaddress_int                    = undef,
   $virtual_ipaddress_excluded               = undef,
   $virtual_routes                           = undef,
+  Boolean $higher_prio_send_advert          = false,
   $smtp_alert                               = false,
   $nopreempt                                = false,
   $preempt_delay                            = undef,
   $advert_int                               = 1,
   $garp_master_delay                        = 5,
   $garp_master_refresh                      = undef,
+  Integer $garp_lower_prio_repeat           = 0,
   $notify_script_master                     = undef,
   $notify_script_backup                     = undef,
   $notify_script_fault                      = undef,
   $notify_script_stop                       = undef,
   $notify_script                            = undef,
+  $notify_script_master_rx_lower_pri        = undef,
```

Comment:
Please add a datatype for this as well. I assume this is a path to a script? In this case, you can use `Stdlib::Absolutepath`. Otherwise, `String[1]` might be correct?

Reply:
notify_script_master_rx_lower_pri is an optional script option for notify_master_rx_lower_pri keepalived parameter. This is a similar option to other notify_script_* parameters already available in  this module where datatype is not defined. Adding the datatype to this parameter makes this option mandatory in hiera, but generally this option is not needed in many environments.

Reply:
In that case the datatype should be `Optional[Stdlib::Absolutepath]`

Reply:
Thanks. I have added it now.